src/DynamoCore/Linting/Interfaces/IRuleEvaluationResult.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
```

Comment:
I think we can remove all the using statements on all files in all files in Interfaces folder.
And since IRuleEvaluationResult, RuleEvaluationStatusEnum & SeverityCodesEnum are very small maybe we can just use a single file.